identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -523,6 +523,7 @@ organization.login=Organization Login
 organization.name=Organization name
 organization.email=Organization email
 invalid.organization.discovery.input=Unable to identify the organization. Please check if the provided email domain is correct or provide the organization name to continue.
+invalid.organization.discovery.type=Given organization discovery type is invalid or not enabled.
```

Comment:
Will the term `organization discovery type` be intuitive to end users?

Reply:
This error message is supposed to be used when multiple organization discovery types (other than email domain) are introduced in the future. If there are multiple organization discovery types, then it can be specified in the authorization request as a query param.

When another discovery type is introduced and the user is allowed to chose the relevant discovery type, then the user will know what `organization discovery type` is.